CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
  - #2846, Fix error when requesting `Prefer: count=<type>` and doing null filtering on embedded resources - @laurenceisla
  - #2959, Fix setting `default_transaction_isolation` unnecessarily - @steve-chavez
  - #2929, Fix arrow filtering on RPC returning dynamic TABLE with composite type - @steve-chavez
+ - #2974, Clean `Response` module off `Wai` leftovers - @develop7
```

Comment:
Oh, we don't keep refactors on the CHANGELOG. Only feats/fixes/breaking changes.
```suggestion
```

Reply:
@develop7 This one still remains

Reply:
yup, not anymore